### ISSUE TITLE
-more- in shops is not quite right

### ISSUE THREAD

**Issue by angband-import** [angband/angband#775]
**Reported by ctate on 9 Jul 2007 07:29 UTC**
Actions after buy/sell incur a superfluous (and quite annoying) 'more' prompt.  Here's the scenario:

Sell something in a shop

```
You sold a Sling (x2) (+0,+0) (l) for 4 gold.
```

Press 'i' to bring up your inventory

```
You sold a Sling (x2) (+0,+0) (l) for 4 gold. -more-
```

Whoops!  The 'more' prompt should not be triggered here; the data about to be displayed is the result of user request, not the continuation of the previous output.

**Comment by angband-import** [comment]
**Modified by ajps on 13 Jul 2007 17:42 UTC**

**Comment by angband-import** [comment]
**Comment by ajps on 13 Jul 2007 18:07 UTC**
Fixed in [335] merged in [5dfe994] (SVN r399).
